### DISCUSSION TITLE
oxc加载文件速度

### DISCUSSION THREAD

**Discussion by huihuang2000** [oxc-project/oxc#7416] [Q&A]
let allocator = Allocator::default();
    let (output, source_type) =  deobfuscator_handle(input_path, &allocator);
    let mut program = Parser::new(&allocator, &output, source_type).parse().program;
这几行代码，耗时得60多ms。所以我想可不可以有更加快的办法，应为需要特别快的速度

**Comment by Boshen** [comment]
The only trick I can provide is to use simdutf8 to validate files. Or skip it if you need extreme speed.

https://github.com/oxc-project/oxc/blob/6f0fe38bff1b727caba3b0a6b31058b80bfef402/crates/oxc_linter/src/utils/mod.rs#L44-L56

Close as lack of more context. And please use English.
